pkg/synchronizer/synchronize-device-group.go

```diff
@@ -79,7 +79,7 @@ func (s *Synchronizer) SynchronizeDeviceGroup(scope *AetherScope, dg *DeviceGrou
 
 	err = validateIPDomain(ipd)
 	if err != nil {
-		return 0, fmt.Errorf("DeviceGroup %s invalid: %s", *dg.DeviceGroupId, err)
+		return 0, fmt.Errorf("DeviceGroup %s invalid IPDomain: %s", *dg.DeviceGroupId, err)
```

Comment:
you could add in the ipd ID too in the error message

Reply:
will do